mongodbatlas/data_source_mongodbatlas_federated_database_instance.go

```diff
@@ -122,6 +122,10 @@ func schemaFederatedDatabaseInstanceDatabasesDataSource() *schema.Schema {
 											Type:     schema.TypeString,
 											Computed: true,
 										},
+										"dataset_name": {
+											Type:     schema.TypeString,
+											Computed: true,
+										},
```

Comment:
do we need to update also the datasource federated database instances?

Reply:
Not required, the datasource federated database instances re-uses the same methods from this data source